config/params.ini

```diff
@@ -222,6 +222,16 @@ corpse_explosion=
 clear_pindle_pack=
 heart_of_wolverine=
 
+[bone_necro]
+teleport=
+teeth=
+bone_armor=
+clay_golem=
+bone_wall=
+bone_spear=
+bone_spirit=
+damage_scaling=1
```

Comment:
also should probably add a comment here relative to what you said in your PR for the users who don't look at PRs. "you can set the damage_scaling option under [bone_necro]. ie: damage_scaling=2 will cast all skills twice as long."

Reply:
Rather than comment the config (technically you can do both I guess), will you update Readme.md? Otherwise lgtm. @slapchop58

Reply:
Sure. I can take care of both those. Those logger comments just got copied from the poison necro implementation that I copied over and modified as I was building this out.